.github/release-drafter.yml

```diff
@@ -39,6 +39,8 @@ categories:
     labels:
       - 'dependencies'
       - 'maintenance'
+  - title: '🖌 Updates in UI'
+    label: 'ui'
```

Comment:
label or labels?

Reply:
I used the same syntax which is used for `documentation` or `apiv2`